### DISCUSSION TITLE
docs: deindex old versions of documentation

### DISCUSSION THREAD

**Discussion by MilanKovacic** [mikro-orm/mikro-orm#5952] [Ideas]
Currently, even very old versions are still indexed by search engines, and create a clutter.
For example, when searching for "mikroorm repository" on Google, in the first 5 results, one of them leads to page from v2.7, other one to page from v4.5.
I believe this can be controlled through "noIndex" property in docusaurus.

**Comment by B4nan** [comment]
The `noIndex` option is a site wide, you cant use that to skip only a part of the docs. I guess we would have to add some custom components to deal with this. 

Btw on my end google gives me only links to latest.

**Reply by B4nan** [reply]
I do get a few results from https://mikro-orm.github.io/docs/v2/repositories/ which is a snapshot of the old documentation, we can surely disable indexing of that, or maybe even remove it completely.

**Reply by MilanKovacic** [reply]
Seems this at least used to be possible: https://github.com/facebook/docusaurus/pull/7963
I also see some mentions here: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-docs#VersionsConfig
I didn't use docusaurus in some time, so I would have to check in detail.

It might also be worth having one versioned docs per major release, so instead of 6.0, 6.1, 6,2.., there would be v6, or 6.x.
Old major (unsupported) versions could be archived as in docusaurus docs:

![image](https://github.com/user-attachments/assets/e822083a-ac5b-4153-a86c-77d8b63bb9f4)

**Reply by B4nan** [reply]
lets see how it works a3d9e5023bcbffaf82e30659fb5a06d5320212dd
